manager/state/raft/raft.go

```diff
@@ -1703,14 +1707,23 @@ func (n *Node) processInternalRaftRequest(ctx context.Context, r *api.InternalRa
 	select {
 	case x, ok := <-ch:
 		if !ok {
+			// Wait notification channel was closed. This should only happen if the wait was cancelled.
+			log.G(ctx).Errorf("Wait cancelled, likely because node %x lost leader position. Wait channel closed with nothing to read.", n.opts.ID)
+			if atomic.LoadUint32(&n.signalledLeadership) == 1 {
+				log.G(ctx).Errorf("Wait cancelled but node %x is still a leader.", n.opts.ID)
```

Comment:
@anshulpundir based on our discussion, let's update these message to remove "wait cancelled".

Reply:
Changed to lowercase. The log above (line 1713) can help us differentiate this from the case below.